target/linux/kirkwood/base-files/etc/board.d/02_network

```diff
@@ -30,9 +30,9 @@ case "$board" in
 	ucidef_add_switch "switch0" \
 		"0:lan" "1:lan" "2:lan" "3:lan" "4:wan" "5@eth0" "6@eth1"
 	;;
-"nsa310b")
+"nsa310b"|"nsa325")
 	ucidef_set_interface_lan "eth0" "dhcp"
-	ucidef_set_interface_macaddr "lan" $( fw_printenv ethaddr | awk -F"=" '{print $2}' )
+	ucidef_set_interface_macaddr "lan" $( mtd_get_mac_ascii uboot_env ethaddr )
```

Comment:
This change has to go into an extra commit since it changes code related to the nsa310b as well. Maybe create a "kirkwood: cleanup nsa310b mac address extraction" commit before the commit adding support for the nsa325.